.github/workflows/persistent_store_test.yml

```diff
@@ -12,20 +12,33 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: thumbv7em-none-eabi
 
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
-          override: true
```

Comment:
By default (i.e. when `override` is not set to true) the action will respect the `rust-toolchain` file. Therefore it shouldn't be necessary to specify we want nightly.
But this requires the git submodules to be checked-out (which you added on this PR)

Reply:
I think it's better to check that the code compiles with both versions which is why I keep nightly.